### DISCUSSION TITLE
请问umi3下如何使用twin.macro？

### DISCUSSION THREAD

**Discussion by wangwufan** [umijs/umi#11085] [Q&A]
有个组件用到了它，但我一直迁移不进去（迁移到umi项目），配置不明白，编译老报错，想请教请教各位老师
![image](https://user-images.githubusercontent.com/12657738/236102003-4cc7bc31-1f5a-4989-903f-c17b7c27c158.png)

**Comment by fz6m** [comment]
umi 3 不维护了，推荐升级到 umi 4

**Reply by wangwufan** [reply]
我知道有umi4，但升级谈何容易

**Reply by DemonSam** [reply]
这个问题有解决吗？我最近也用twin.macro  ，umi4创建，也不行

**Comment by wangwufan** [comment]
> 这个问题有解决吗？我最近也用twin.macro ，umi4创建，也不行

我放弃umi了，用vite，速度很快，打包体积很小，脚手架相关的功能可以用其他包代替
